test/format/test_speedscope.py

```diff
@@ -232,4 +233,4 @@ def test_speedscope_wall_metrics_only():
         speedscope.dump(text_stream)
 
         with open(join(dirname(DATA_FILE), "austin.json"), "r") as sprof:
-            assert text_stream.getvalue() == sprof.read()
+            assert text_stream.getvalue() == json.dumps(json.load(sprof))
```

Comment:
I think we should be able to keep this as
```suggestion
            assert text_stream.getvalue() == sprof.read()
```
if we change line 224 above into
~~~ python
speedscope = Speedscope("austin.out", mode, indent=2)
~~~

Reply:
Done in [`f09f13ae`](https://github.com/P403n1x87/austin-python/pull/7/commits/f09f13ae93e3ad2c0b3e71189438f7f14a6dff15).